packages/api/src/webhooks/cannon.ts

```diff
@@ -52,7 +52,70 @@ export default class WebhookCannon {
 
   async start() {
     console.log("WEBHOOK CANNON STARTED");
-    await this.queue.consume(this.handleQueueMsg.bind(this));
+    await this.queue.consume("webhooks", this.handleQueueMsg.bind(this));
+    await this.queue.consume("events", this.handleEventsQueue.bind(this));
+  }
+
+  async handleEventsQueue (data: ConsumeMessage) {
+    let message;
+    try {
+      message = JSON.parse(data.content.toString());
+      console.log("events: got message", message);
+    } catch (err) {
+      console.log("events: error parsing message", err);
+      this.queue.ack(data);
+      return;
+    }
+    let event = message;
+
+    try {
+      const { data: webhooksList } = await this.db.webhook.listSubscribed(
+        event.userId,
+        event.event
+      );
+  
+      console.log("webhooks : ", webhooksList);
+      let stream = await this.db.stream.get(event.streamId);
+      if (!stream) {
+        // if stream isn't found. don't fire the webhook, log an error
+        return console.error(
+          `webhook Cannon: onTrigger: Stream Not found , streamId: ${event.streamId}`
+        );
+      }
+      // basic sanitization.
+      let sanitized = this.db.stream.addDefaultFields(
+        this.db.stream.removePrivateFields({ ...stream })
+      );
+      delete sanitized.streamKey;
+  
+      let user = await this.db.user.get(event.userId);
+      if (!user) {
```

Comment:
Maybe check if user is suspended here well?
```suggestion
      if (!user || user.suspended) {
```